.claude/skills/nextjs-page-migration/SKILL.md

```
@@ -86,6 +86,91 @@ Phase A–F で 18 枚のガイドページが `web-next/` App Router に**全
+/* ✅ デスクトップのデフォルトを先に定義 */
+.sidebarToggle { display: none; }
+
+@media (max-width: 960px) {
+  .sidebarToggle { display: flex; /* ... */ }
+}
+
+/* ❌ NG: @media 内にしか書かない → デスクトップでも表示されてしまう */
+```
+
+**確認コマンド（var() 参照の棚卸し）**:
+```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)